test/recipes/80-test_ssl_old.t

```diff
@@ -416,29 +416,30 @@ sub testssl {
             push @exkeys, "-s_cert", "certE.ss", "-s_key", "keyE.ss";
         }
 
-	my @protocols = ();
+	my %protocols = ();
 	# FIXME: I feel unsure about the following line, is that really just TLSv1.2, or is it all of the SSLv3/TLS protocols?
-        push(@protocols, "TLSv1.3") unless $no_tls1_3;
-        push(@protocols, "TLSv1.2") unless $no_tls1_2;
-        push(@protocols, "SSLv3") unless $no_ssl3;
-	my $protocolciphersuitcount = 0;
-	my %ciphersuites =
-	    map { my @c =
-		      map { split(/:/, $_) }
-		      run(app(["openssl", "ciphers", "${_}:$ciphers"]),
-                          capture => 1);
-		  map { s/\R//; } @c;  # chomp @c;
-		  $protocolciphersuitcount += scalar @c;
-		  $_ => [ @c ] } @protocols;
+	$protocols{"TLSv1.3"} = 1 unless $no_tls1_3;
+	$protocols{"TLSv1.2"} = 1 unless $no_tls1_2;
+	$protocols{"SSLv3"} = 1 unless $no_ssl3;
+	my $protocolciphersuitecount = 0;
+	my %ciphersuites = ();
+	foreach (map { s|\R||; $_ }
+		 run(app(["openssl", "ciphers", "-v", "ALL:$ciphers"]), capture => 1)) {
```

Comment:
A comment here.  @ekasper, I know I talked about changing `run` to return both the success / failure status and the list of lines and handling that here.  If you don't mind, I'll push that to a PR after this one has been merged.  It makes the job easier for me.